R/correlation.R

```diff
@@ -9,10 +9,12 @@
 #'   to compute for all numeric variables in data.
 #' @param method a character string indicating which correlation coefficient
 #'   is to be computed. One of "pearson" (default), "kendall", or "spearman"
-#' @param partial Logical indicating whether the pairwise partial correlation
-#' for three variables should be computed. Defaults to `FALSE`. If set to `TRUE`
-#' all combinations, i.e. pairs, of specified variables will be computed
-#' while controlling for the third variable.
+#' @param partial Either a logical value or a character string. When set
+#' to `TRUE`, the function computes the pairwise partial correlation for three
+#' variables. If set to a character, it represents the variable to control for.
```

Comment:
The rest of the package allows sym links (just like every function in the tidyverse). In this regard, I would also opt here for sym-link possibility:
```r 
WoJ %>% 
  correlate(autonomy_selection, autonomy_emphasis, partial = work_experience)
```

Reply:
This would then also require a slight adjustment in the docs.